plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -205,12 +204,6 @@ func (l *persistentVolumeLabel) findVolumeLabels(volume *api.PersistentVolume) (
 			return nil, fmt.Errorf("error querying GCE PD volume %s: %v", volume.Spec.GCEPersistentDisk.PDName, err)
 		}
 		return labels, nil
-	case volume.Spec.AzureDisk != nil:
-		labels, err := l.findAzureDiskLabels(volume)
-		if err != nil {
-			return nil, fmt.Errorf("error querying AzureDisk volume %s: %v", volume.Spec.AzureDisk.DiskName, err)
-		}
-		return labels, nil
```

Comment:
I am somewhat torn about this. PV labeling is not a feature of the volume plugin per se. I would probably keep the labeling working and remove it once the whole Azure cloud provider is removed, together with AzurFile plugin.

Reply:
reverted